libraries/standard/http/filePaths.cmake

```diff
@@ -0,0 +1,23 @@
+# This file is to add source files and include directories
+# into variables so that it can be reused from different repositories
+# in their Cmake based build system by including this file.
+#
+# Files specific to the repository such as test runner, platform tests
+# are not added to the variables.
+
+# HTTP library source files.
+set( HTTP_SOURCES
+     "${MODULES_DIR}/standard/http/src/http_client.c"
+     "${MODULES_DIR}/standard/http/src/http_client_parse.c" )
+
+# HTTP library Include directories.
+set( HTTP_INCLUDE_PUBLIC_DIRS
+     "${MODULES_DIR}/standard/http/include"
+     "${MODULES_DIR}/standard/utilities/include"
+     "${ROOT_DIR}/platform/include"
```

Comment:
Can you please remove the platform include directory from this variable and handle separately? This will not be the same path when reused in different repositories.